### ISSUE TITLE
Deal with stock areas

### ISSUE THREAD

**Issue by anhegofcherek** [prophecymud/area-issues#11]
We have a lot of fabulously detailed areas that have been lovingly hand-crafted.

We still have one or two stock areas that are very jarring in how they don't fit (goblins and dragons and ogres oh my!)

They're not easy to just remove because they link places together. The relinking project got rid of most of them, but that was never finished.

My proposal is to do a minimum-viable reskin

1. Delete the extraneous bits - mobs, rooms that aren't routes to other places
2. Flag them noaquest (so legacy objects don't show up)
3. Do a hack job on strings so they're not _completely_ out of place ("The dark dwarf forest" becomes "The dark scary forest" etc.)
4. Hand them over for someone to do a better/more nuanced job of, akin to Slaver's Cove or Tol Rane

The areas in question are:

* Haon Dor
* Marsh
* Miden'nir
* Thalos

**Comment by Vordai-of-Prophecy** [comment]
Absolutely. Rip 'em out by the roots. They're lowering the property values. Pave them over and call them "North/South/East/West (as applicable) Old Wagon Trail" or some such until something original can replace them. Please!

**Comment by anhegofcherek** [comment]
Thalos and Marsh done!

**Comment by anhegofcherek** [comment]
Haon Dor and Miden'nir done, and all deployed and live!

They're not pretty, but they're *better*!
